core/store/orm/orm.go

```diff
@@ -318,7 +336,7 @@ func (orm *ORM) LinkEarnedFor(spec *models.JobSpec) (*assets.Link, error) {
 		Joins("JOIN job_specs ON job_runs.job_spec_id = job_specs.id").
 		Where("job_specs.id = ? AND job_runs.status = ? AND job_runs.finished_at IS NOT NULL", spec.ID, models.RunStatusCompleted)
 
-	if dbutil.IsPostgres(orm.DB) {
+	if dbutil.IsPostgresV2(orm.DB) {
```

Comment:
We havent support SQLite for a year I think its safe to remove this if now :trollface: